packages/core/test/unit/query-generator/truncate-table-query.test.ts

```diff
@@ -0,0 +1,103 @@
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { createSequelizeInstance, expectsql, sequelize } from '../../support';
+
+const dialect = sequelize.dialect;
+
+describe('QueryGenerator#truncateTableQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  it('produces a TRUNCATE TABLE query for a table', () => {
+    expectsql(() => queryGenerator.truncateTableQuery('myTable'), {
+      default: `TRUNCATE [myTable]`,
+      mssql: 'TRUNCATE TABLE [myTable]',
+      sqlite: ['DELETE FROM `myTable`;'],
```

Comment:
This is because sqlite executes two statements if `restartIdentity` is set to true, therefore requiring the semicolon as a separator between the statements.

Reply:
Oh you changed it back to a single string. Ok :)